lib/tasks/grda_warehouse.rake

```diff
@@ -320,11 +320,7 @@ namespace :grda_warehouse do
       Rake::Task['driver:hmis_external_apis:export:ac_clients'].invoke
     end
 
-    if HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists? && RailsDrivers.loaded.include?(:hmis_external_apis)
-      HmisExternalApis::ConsumeExternalFormSubmissionsJob.new.perform(
-        encryption_key: HmisExternalApis::ExternalForms.external_forms_shared_key,
-      )
-    end
+    HmisExternalApis::ConsumeExternalFormSubmissionsJob.new.perform if HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists? && RailsDrivers.loaded.include?(:hmis_external_apis)
```

Comment:
Unfortunately rubocop's `Style/IfUnlessModifier` cop makes this more difficult to read than multi-line conditional

Reply:
Feel free to add a `rubocop:disable Style/IfUnlessModifier` around something if you think it's easier to read in a different format.  The style guide is meant as a guide, but there are definitely situations where we should preference readability over consistency.